tests/azure/templates/playbook_tests.yml

```diff
@@ -0,0 +1,61 @@
+
+parameters:
+  - name: group_number
+    type: number
+    default: 1
+  - name: number_of_groups
+    type: number
+    default: 1
+  - name: scenario
+    type: string
+    default: centos-8
+  - name: ansible_version
+    type: string
+    default: ">=2.9,<2.10"
+  - name: python_version
+    type: string
+    default: 3.6
+  - name: build_number
+
+jobs:
+- job: RunTests${{ parameters.group_number }}
+  displayName: Run tests ${{ parameters.group_number }}/${{ parameters.number_of_groups }}
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '${{ parameters.python_version }}'
+
+  - script: |
+      pip install \
+        "molecule[docker]>=3" \
+        "ansible${{ parameters.ansible_version }}" \
+        pytest \
+        pytest-split-tests
+    displayName: Install dependencies
+
+  - script: |
+      mkdir -p ~/.ansible/roles ~/.ansible/library ~/.ansible/module_utils
+      cp -a roles/* ~/.ansible/roles
+      cp -a plugins/modules/* ~/.ansible/library
```

Comment:
You can also directly copy the plugins folder, that will work also. ~/.ansible/plugins